src/ngx_http_lua_initworkerby.c

```diff
@@ -40,10 +40,33 @@ ngx_http_lua_init_worker(ngx_cycle_t *cycle)
 
     lmcf = ngx_http_cycle_get_module_main_conf(cycle, ngx_http_lua_module);
 
-    if (lmcf == NULL
```

Comment:
I think we should still keep this original `if` statement to make the code below easier to understand.